app/lib/features/news/widgets/news_post_editor/selected_action_button.dart

```diff
@@ -30,13 +30,13 @@ class SelectedActionButton extends ConsumerWidget {
   Widget build(BuildContext context, WidgetRef ref) {
     if (refDetails == null) return SizedBox();
     final refObjectType = refDetails!.typeStr();
```

Comment:
we have checked possible ```null``` case for ```refDetails``` already above, its good to use ```?``` to avoid UI exception :)